codegen/smithy-dafny-codegen-test/src/test/java/software/amazon/polymorph/smithygo/GoTestModels.java

```diff
@@ -31,14 +31,24 @@ class GoTestModels extends TestModelTest {
     DISABLED_TESTS.add("aws-sdks/kms-lite");
     DISABLED_TESTS.add("aws-sdks/sqs");
     DISABLED_TESTS.add("aws-sdks/sqs-via-cli");
+    //TODO: We should ne able to support below models.
+    DISABLED_TESTS.add("MultipleModels");
+    DISABLED_TESTS.add("LocalService");
+
+    //TODO: Pending PR Merge
+    DISABLED_TESTS.add("Positional");
+
+    //TODO: Fix this before main merge
+    DISABLED_TESTS.add("Dependencies");
+
     //V1 Tests are not supported in Go
     DISABLED_TESTS.add("aws-sdks/ddb");
     DISABLED_TESTS.add("aws-sdks/kms");
   }
 
   @ParameterizedTest
   @MethodSource("discoverTestModels")
-  void testModelsForJava(String relativeTestModelPath) {
+  void testModelsForGo(String relativeTestModelPath) {
```

Comment:
non blocking because this function is just to run make command and disabled test.
```suggestion
  void testModelsForGo(final String relativeTestModelPath) {
```